### PR TITLE
ref(integrations): cleans up integration directory code

### PR DESCRIPTION
**Pull request by scefali**
This PR cleans up some of the TODO items related to the integration directory.
1. Removing the unused `installedIntegration.tsx ` and replacing it with the `installedIntegrationInDirectory.tsx` component
2. Moving the `UninstallAppButton` component to the only place where it is used (sentryAppDetailedView)

### CODE REVIEW

File: src/sentry/static/sentry/app/views/organizationIntegrations/installedIntegration.tsx [hunk 1]
```diff
@@ -1,19 +1,19 @@
-import PropTypes from 'prop-types';
```
Comment (scefali) [inline src/sentry/static/sentry/app/views/organizationIntegrations/installedIntegration.tsx:1]: You can ignore the diffs in the file. I just copied `installedIntegrationInDirectory.tsx` into here with the only modification being to remove the TODO item.

File: src/sentry/static/sentry/app/views/organizationIntegrations/installedIntegrationInDirectory.tsx [hunk 2]
```diff
@@ -1,274 +0,0 @@
...
-  onDisable: (integration: Integration) => void;
-  onReinstallIntegration: (integration: Integration) => void;
-  trackIntegrationEvent: (
-    options: Pick<SingleIntegrationEvent, 'eventKey' | 'eventName'>
-  ) => void; //analytics callback
-  className?: string;
-  showReauthMessage: boolean;
-};
-
-//TODO: Rename to InstalledIntegration when we can remove the old one
```
Comment (scefali) [inline src/sentry/static/sentry/app/views/organizationIntegrations/installedIntegrationInDirectory.tsx:34]: This is the TODO item being addressed

**Comment by NisanthanNanthakumar** [comment]
> This PR cleans up some of the TODO items related to the integration directory.
> 
> 1. Removing the unused `installedIntegration.tsx ` and replacing it with the `installedIntegrationInDirectory.tsx` component

I think you mean replacing `installedIntegrationInDirectory.tsx` with `installedIntegration.tsx `, right?

**Comment by scefali** [comment]
@NisanthanNanthakumar I meant moving the code from `installedIntegrationInDirectory` into `installedIntegration` and thereby deleting `installedIntegrationInDirectory`.

**Comment by mgaeta** [comment]
I'd also delete `src/sentry/static/sentry/app/views/organizationIntegrations/shared/styledComponents.tsx`

**Review by NisanthanNanthakumar — APPROVED** [review]

**Review by mgaeta — APPROVED** [review]

**Pull request merged**